### DISCUSSION TITLE
Show all standard projects at all time?

### DISCUSSION THREAD

**Discussion by motoyugota** [bafolts/terraforming-mars#2841] [General]
Hey everyone, I am just curious as to the reasoning behind this decision in the UI. Especially the reasoning of it becoming the selected option when you don't have enough money for any of them. If you can't play any of your cards, the "Play a card" option is not available to you. And when it is available, it only shows you the cards you can play at that point, not all of them. Why are the standard projects any different here?

**Comment by nwai90** [comment]
@motoyugota This may be changed at some point, there is a consideration to move SP costs into help page since it is purely informative

**Comment by vsrisuknimit** [comment]
Standard project costs are already available here: https://terraforming-mars.herokuapp.com/help.
No need to show the information when just a line below you can already see all cards in hand. Showing all cards again in 'Select card to play' would be information overload.

**Reply by motoyugota** [reply]
Get rid of the "cards in hand" and just combine it with "play a card". It's dumb to have both like that anyways.

**Reply by vsrisuknimit** [reply]
It has been talked about. But no one is working on it yet. https://github.com/bafolts/terraforming-mars/discussions/2121

**Comment by kevinb9n** [comment]
slight background: I think it is a very good *general* principle to keep options fully visible which are commonly available but just not *currently* available. I've pushed that idea here, because it helps with planning, even for players who do technically *know* the information, because have only so much brain to use at any time.

HOWEVER, that principle was kinda assuming that the options are present in some *unobtrusive* way. Once they're getting in people's way, the equation may shift a bit. That was my bad, since I was mentally picturing some more long-term design directions and not how it really is today.

**Comment by kberg** [comment]
Standard projects are now only available in learner mode. Closing this one.

**Reply by motoyugota** [reply]
> Standard projects are now only available in learner mode. Closing this one.

What in the world are you talking about? Standard projects are a normal part of the game. Are you confusing standard projects with the standard corporations for some reason? 

This should not have been closed.

**Comment by kberg** [comment]
Let me expand my overshort incomplete sentence. In learner mode, standard projects are shown even if none are available. With learner mode disabled, if a player cannot take a standard project, the list is not shown.

In my opinion, closing it was the correct choice. Unless I'm mistaken, there's nothing new to add to the discussion. Please let me know if you have something new.
